www/ios/Entry.js

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+*/
+
+module.exports = {
+    /**
+   * Return a URL that can be used to identify this entry.
+   * Use a URL that can be used to as the src attribute of a <video> or
+   * <audio> tag. If that is not possible, construct a http(s)://(localhost) URL.
+   */
+    toURL: function () {
+        return window.location.origin.includes('file://')
+            ? this.nativeURL
+            : window.WkWebView.convertFilePath(this.nativeURL);
```

Comment:
If using cordova-ios < 6 and some WebView engine plugin that allowed using schemes before cordova-ios did, it will call `convertFilePath`, which won't be available and will throw and error.
We should probably require cordova-ios >= 6 in the engines for being sure we can use this method.

Also the logic could be simplified since `convertFilePath` will already return the `file://` path if not using a scheme, so could just call convertFilePath.

```suggestion
        return window.WkWebView.convertFilePath(this.nativeURL);
```

Reply:
This plugin doesn't seem to declare a minimum required cordova-ios version, so we could definitely add >= 6.0.0 and do a major version bump.

If we prefer to do this without a major bump, checking for the existence of  WKWebView wouldn't be too bad:
```js
return (window.WkWebView && window.WkWebView.convertFilePath)
    ? return window.WkWebView.convertFilePath(this.nativeURL)
    : this.nativeURL;
```

Reply:
> If we prefer to do this without a major bump, checking for the existence of WKWebView wouldn't be too bad:

The intent was to hopefully fix the API for iOS so that the file entry API can be used the same way for both android and iOS and release this fix in 8.1.3 patch release, so I think this is the best suggestion.

Later we can simplify to jcesar's suggestion with a cordova-ios >= 6 requirement

Reply:
Tested the change on iOS 15 in the following configurations:

Cordova iOS 6.x
- With schemes
- Without schemes

Cordova iOS 5.x
- Without schemes

So it seems like this retains the backwards compatibility as intended.